dbt_sugar/core/task/doc.py

```diff
@@ -56,10 +58,123 @@ def run(self) -> int:
         return 0
 
+    def change_model_description(self, content: Dict[str, Any], model_name: str) -> Dict[str, Any]:
+        """Updates the model description from a model.
+
+        Args:
+            content (Dict[str, Any]): content of the schema.yml.
+            model_name (str): model name to change the description from.
+            description (str): description of the model.
+
+        Returns:
+            Dict[str, Any]: with the schema.yml updated.
+        """
+        model_doc_payload: List[Mapping[str, Any]] = [
+            {
+                "type": "confirm",
+                "name": "wants_to_document_model",
+                "message": "Model Description: This is my previously documented model. Document?",
+                "default": True,
+            },
+            {
+                "type": "text",
+                "name": "model_description",
+                "message": "Please write down your description:",
+            },
+        ]
+        user_input = UserInputCollector("model", model_doc_payload).collect()
+        if user_input.get("wants_to_document_model", False):
+            for model in content.get("models", []):
+                if model["name"] == model_name:
+                    model["description"] = user_input["model_description"]
+        return content
+
+    def find_model_in_dbt(self, model_name: str) -> Tuple[Optional[Path], bool]:
+        """
+        Method to find a model name in the DBT depository.
+
+            - If we find the sql of the model but there is no schema we return the Path
+            and False (to create the schema).
+            - If we find the sql of the model and there is schema we return the Path and True.
+
+        Args:
+            model_name (str): model name to find in the DBT repository.
+
+        Returns:
+            Tuple[Optional[Path], bool]: with the path of the sql model
+            and the boolean indicating if the schema.yml exists.
+        """
+        for root, _, files in os.walk(self.repository_path):
+            for file in files:
+                # TODO: Check how to avoid using target to discriminate compiled files in DBT.
+                if file == f"{model_name}.sql" and "target" not in root:
+                    path_file = Path(os.path.join(root, "schema.yml"))
+                    if path_file.is_file():
+                        return path_file, True
+                    else:
+                        return path_file, False
+        return None, False
+
+    def orchestrator_document_model(self, model_name: str, columns_sql: List[str]) -> None:
+        """
+        Orchestrator to fully document a model will:
+
+            - Create or update the columns from the database into the shema.yaml.
+            - Gives the user the posibility to change the model description.
+            - Gives the user the posibility to document the undocumented columns.
+            - Updates all the new columns definitions in all DBT.
```

Comment:
Not sure I follow what you mean with that last point maybe try reformulating what you want a developer to understand that would have never seen this code.

That being said the fact that you do these nice summaries is already extremely helpful! 👌 
```suggestion
            - Updates all the new columns definitions in all dbt.
```